__tests__/api/chat/channels/read.test.js

```diff
@@ -45,28 +45,6 @@ describe('POST /chat/channels/read', () => {
     );
   });
 
-  test('should return 403 error validation if channel type is not provided', async () => {
-    const response = await supertest(app)
-      .post('/api/v1/chat/channels/read')
-      .set({Authorization: `Bearer token`})
-      .send({
-        channelId: 'channel-id'
-      })
-      .expect(403);
-
-    expect(response.body).toEqual(
-      expect.objectContaining({
-        status: 'error validation',
-        message: expect.arrayContaining([
-          expect.objectContaining({
-            field: 'channelType',
-            message: expect.any(String)
-          })
-        ])
-      })
-    );
-  });
-
   test('should return 403 error validation if channelType is not chat or group', async () => {
     const response = await supertest(app)
       .post('/api/v1/chat/channels/read')
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Removing the channelType checking in this test might lead to less coverage of potential error scenarios. If the API endpoint `/api/v1/chat/channels/read` still requires a `channelType`, it would be better to keep this test to ensure that the validation is working correctly. If the requirement for `channelType` has been removed from the endpoint, then this change is fine.
```diff
-    test('should return 403 error validation if channelType is not chat or group', async () => {
-      const response = await supertest(app)
-        .post('/api/v1/chat/channels/read')
```
If the `channelType` is still required, consider modifying the test to reflect the new requirements.


<!-- This is an auto-generated comment by OSS CodeRabbit -->